### ISSUE TITLE
Validation loss does not decrease

### ISSUE THREAD

**Issue by scogna02** [mahmoodlab/CLAM#189]
Hi everyone!
Thank you for such a great framework! 
I'm trying to apply it to different WSI tissues and my dataset has only around 130 images, but after I apply features extraction
and train the model, my validation loss starts increasing drastically after only 10 epochs. I tried tuning all the possible parameters but that didn't change much. If you have any suggestions I'll be really happy!

**Comment by fedshyvana** [comment]
Hey! if your dataset is super small and potentially imbalanced, you'll probably have a hard time getting good performance. I think your best bet might be to use self-supervised learning/domain-specific transfer learning from other pretrained encoders instead of using the ResNet50 encoder suggested by this repo (which is quite outdated).

**Comment by scogna02** [comment]
Hi! First thank you for your kind response. I wanted to ask you if you have any ideas of other pertained encoders that you think might fit better to my task. Thank you again!
